andela/bp-esa-frontend
# #164138244 Status Icon Alignment and Font of the Application

#### What does this PR do?
Makes design changes outlined in the tasks below.

#### Description of Task to be completed?
- Fix alignment of status info icon
- Rename wrongly named variables
- Change font of the application to 'Din Pro'
- Make the automation details modal scrollable

#### How should this be manually tested?
1. Clone the repo `https://github.com/andela/bp-esa-frontend`
2. Check out to `bp-esa-frontend` folder and run this command on your terminal `git fetch && git checkout bg-make-font-change-164138244` and run `yarn install` on your terminal
3. Start the application with the command `yarn start`

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[164138244](https://www.pivotaltracker.com/story/show/164138244)

#### Postman Documentation
N/A

#### Screenshots (If applicable)

<img width="1440" alt="screen shot 2019-02-21 at 20 41 00" src="https://user-images.githubusercontent.com/29487740/53189708-2edf9e00-3619-11e9-95b4-81d13cb00500.png">

![scroll2](https://user-images.githubusercontent.com/29487740/53252552-3d3cc100-36d0-11e9-9879-15dcac3e9ab9.gif)
